pkg/ebpf/config.go

```diff
@@ -84,11 +84,6 @@ func NewDefaultConfig() *Config {
 func (c *Config) EnabledKProbes() map[KProbeName]struct{} {
```

Comment:
Let's add to the comment above that this doesn't include kprobes used for offset guessing

Reply:
👍